### PR TITLE
First notebook, edited at and pushed from jupyterlab

### PR DESCRIPTION
**Pull request by drphilmarshall**
@kadrlica Here's an example tutorial notebook, for teaching Stack Club members a sensible git/GitHub workflow. I made this in and pushed it from the jupyterlabdemo :-) See what you think!

This is in response to #11

### CODE REVIEW

**Comment by drphilmarshall** [comment]
PS. For a GitHub rendering of this Notebook, see [here](https://github.com/LSSTScienceCollaborations/DMStackClub/blob/issue/11/hello-world-pjm/notebooks/Hello_World.ipynb).

**Comment by kadrlica** [comment]
Tried to clone from jupyterlabdemo. Got:

```
[kadrlica@jld-lab-kadrlica-r150 ~]$ git clone git@github.com:LSSTScienceCollaborations/DMStackClub.git
Cloning into 'DMStackClub'...
The authenticity of host 'github.com (192.30.253.113)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
RSA key fingerprint is MD5:16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added 'github.com,192.30.253.113' (RSA) to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.```

**Comment by kadrlica** [comment]
RE: Contribution structure. From this I gather that all notebooks will be stored in github as "clean" (meaning no output) notebooks. Will we also have a resource that runs all the notebooks are shows the output as a resource for people who are reading through but not executing?

**Comment by drphilmarshall** [comment]
That's what I was thinking, yeah - have travis run the notebooks, `nbconvert` them to html, and deploy them back to the repo.

File: notebooks/Hello_World.ipynb [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "source": [
+    "## Requirements\n",
+    "You need to be looking at this notebook from within a `jupyterlab` environment. If you are, that means you have successfully started a terminal via the Launcher, and cloned the DMStackClub repo via `git clone git@github.com:LSSTScienceCollaborations/DMStackClub.git` (in this case, after adding your public SSH key to your GitHub profile)."
```
[the remark this answers is not in the document]
Comment (kadrlica) [reply notebooks/Hello_World.ipynb]: I would change this (and other instructions in this tutorial) to use https. It seems that the GitHub authentication does not require a password be provided for `clone` and `push` with https.

**Comment by kadrlica** [comment]
I was able to follow the instructions to create `issue/11/hello-world-adw`.

**Comment by drphilmarshall** [comment]
I'm going to go ahead and merge this PR, and then also merge in Alex's improvements. Thanks @kadrlica !

**Pull request merged**